use

```diff
@@ -0,0 +1,228 @@
+mysql  Ver 15.1 Distrib 10.6.12-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2
```

Comment:
Da li je ovo trebalo da bude deo PR-a?

Reply:
Mislim na ceo fajl.

Reply:
Ovo je nesto vase, intellij ga je auto pokupio. Predpostavljam da je nesto vezano da docker, ali ga ja sigurno nisam sam iskucao

Reply:
Obrisacu ga ali to je samo file sa listom komandi